eng/Versions.props

```diff
@@ -138,6 +138,8 @@
     <SystemThreadingAccessControlVersion>7.0.0</SystemThreadingAccessControlVersion>
     <runtimenativeSystemIOPortsVersion>10.0.0-preview.3.25152.4</runtimenativeSystemIOPortsVersion>
     <!-- Keep toolset versions in sync with dotnet/msbuild and dotnet/sdk -->
+    <!-- The framework targeted by and package release band for the minimum toolset version our tasks must support -->
+    <ToolsetTargetFramework>net8.0</ToolsetTargetFramework>
```

Comment:
This is essentially `NetToolMinimum`: https://github.com/dotnet/arcade/blob/63c161539bde4a68f27bc66598137da578331d85/src/Microsoft.DotNet.Arcade.Sdk/tools/TargetFrameworkDefaults.props#L37-L40

Reply:
I wondered if we already had it somewhere.  Let's hold off on changing this PR and burning more build resources untilw we understand what direction we want to take.  I'm not sure if this one is going to pan out.